os/drivers/bch/bchlib_read.c

```diff
@@ -149,6 +149,11 @@ ssize_t bchlib_read(FAR void *handle, FAR char *buffer, size_t offset, size_t le
 	 * into the user buffer.
 	 */
 	if (len >= bch->sectsize) {
+		if (bch->sectsize <= 0) {
```

Comment:
This variable is used at other places in the lines above in similar checks and calculations.
Should it be checked in the very beginning of this function?

Maybe some logic tightening is required to curb the very source of the negative values for this variable?

Reply:
Done. Thanks